test/e2e/framework.go

```diff
@@ -1278,6 +1293,34 @@ func (data *TestData) deployFlowVisibilityCommon(chOperatorYML, flowVisibilityYM
 	if err != nil || rc != 0 {
 		return fmt.Errorf("error when deploying the ClickHouse Operator YML: %v\n is %s available on the control-plane Node?", err, chOperatorYML)
 	}
+	// Check for clickhouse operator pod running again for db connection establishment
```

Comment:
Maybe we can check the deployment directly? 
https://github.com/antrea-io/antrea/blob/9c32416d85a919f0ab087e5b564afe808249295a/test/e2e/framework.go#L1084

Reply:
Thanks! That makes sense to me. But I found simply check the ClickHouse Operator is not enough and removed these parts. For more details, you can see my comments below